resources/public/api/conf/1.0/application.yaml

```diff
@@ -166,20 +173,13 @@ paths:
       summary: Submit a message for an excise movement.
       description: |
         Submit a message for a specific excise movement. </br></br>
-        **Existing SOAP API equivalents:** 
-        + SubmitReportofReceipt
-        + SubmitExplainDelayToDelivery
-        + SubmitCancellation
-        + SubmitChangeOfDestination
-        + SubmitAlertOrRejectionMovement
-        + SubmitSplitMovement
-        + SubmitReasonForShortage
-        
         The **X-Message-Type** header **must** be the same as the message body e.g. for an **IE818 (Record of receipt) message** the **X-Message-Type** header also needs to be **IE818**
         
-        **Note:** The consignor ID or consignee ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
+        **Note:** The consignor ID or consignee ID in the message must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
```

Comment:
Not sure how best to word this but 813 and 810 don't have the consignor or (current) consignee in the message itself, so we validate it against the value from the database.

Reply:
These messages will have a ARC though right? We should be able to pull out the movement using the ARC and then check if one of their ERNs match what we have recorded in the movement (I know there is a question around IE813 and whether that means we either add an additional consignee to the movement that needs addressing too)

Reply:
yeah that's what we do, take the arc, look up consignor/consignee in database.

Reply:
OK to leave this as-is? Hopefully the general idea is there, if some messages don't have the consignor/consignee not sure of a better way to explain